android/app/src/androidTest/java/com/react/testapp/ManifestTest.kt

```diff
@@ -0,0 +1,69 @@
+package com.react.testapp
+
+import com.react.testapp.manifest.Manifest
+import com.react.testapp.manifest.ManifestJsonAdapter
+import com.react.testapp.manifest.MoshiBundleAdapter
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+
+class ManifestTest {
+    private lateinit var adapter: JsonAdapter<Manifest>
+
+    @Before
+    fun setUp() {
+        adapter = ManifestJsonAdapter(
+            Moshi.Builder()
+                .add(MoshiBundleAdapter())
+                .build()
+        )
+    }
+
+    @Test
+    fun parsesMultipleComponentsWithProps() {
+        val json = """
```

Comment:
Can we add an issue to consolidate all the manifest parsing tests?

Reply:
Do you refer to what we have discussed during the last sync?

Reply:
Yes

Reply:
https://github.com/microsoft/react-native-test-app/issues/193

Reply:
Thanks ❤️